src/commit.rs

```diff
@@ -1 +0,0 @@
-
```

Comment:
I guess this file was added way to early. So this file should not show in the git commit history..

Reply:
At the beginning, I thought giving every data structure a module on its own would be a good idea.  But then, I read in the Rust book that this is what `src/lib.rs` is usually used for.

Reply:
Yeah, it is totally normal to have multiple structs in one file or module, different to java, c++ or c#.